ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-aro.yaml

```diff
@@ -0,0 +1,96 @@
+base_images:
+  ansible:
+    name: "4.11"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.11"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.11"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.11"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  intermediate410:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+  latest:
```

Comment:
Seem like this release image is never used in the following step, neither is not needed in aro-provision-cluster, right?

Reply:
That is correct, and if I try and remove the unused releases:latest I get this validation failure when running make jobs:

docker run --user 0:0 --platform linux/amd64 --rm -v "/home/m/git-local/openshift/mgahagan73/release/ci-operator/config:/ci-operator/config:z" -v "/home/m/git-local/openshift/mgahagan73/release/ci-operator/step-registry:/ci-operator/step-registry:z" registry.ci.openshift.org/ci/ci-operator-checkconfig:latest --config-dir /ci-operator/config --registry /ci-operator/step-registry
time="2023-03-09T19:08:17Z" level=error error="failed to validate configuration /ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly-4.11-upgrade-from-aro.yaml: invalid configuration: tests[0].literal_steps.test[1].dependencies[0]: cannot determine source for dependency \"release:latest\" - this dependency requires a \"latest\" release, which is not configured"

Reply:
Per the error message, some steps in `openshift-upgrade-qe-test` chain has dependencies setting for `release:latest`.  Let us keep it. But per my understanding, generally the `release:latest` indicated the source version, so I suggest to update its value to 4.10 nightly instead of 4.11 nightly.

Reply:
@mgahagan73 I guess ARO is using some stable build, though we can not determinate which version to be sued in ARO install, to be closer to the source version, I suggest to point `release:latest` to stable build instead of nightly build.

Reply:
[dedf5c0](https://github.com/openshift/release/pull/36814/commits/dedf5c0f78cb6c55f3df45f3da1526fa0889b92c)

Reply:
we are supposed to use **stable** build instead of **nightly** build, right?